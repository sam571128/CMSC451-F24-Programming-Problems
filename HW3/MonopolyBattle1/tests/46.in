45 1596 49
14 11
32 12
26 26
21 4
44 28
26 18
24 14
12 8
19 31
43 15
27 26
16 1
43 18
6 21
37 40
27 28
31 7
11 45
28 31
32 11
40 33
32 21
43 35
39 40
37 36
29 45
15 43
13 11
45 19
21 25
32 8
34 5
15 42
28 6
42 30
10 32
31 38
36 27
14 27
31 5
6 42
19 37
26 25
25 27
2 16
36 23
3 14
19 21
30 12
20 7
37 13
19 11
30 3
14 39
38 22
44 25
26 39
9 38
5 43
23 11
29 13
1 25
45 27
26 7
21 33
38 37
37 23
43 8
33 24
26 32
37 26
45 37
34 32
23 37
5 12
17 27
29 10
38 44
19 30
16 45
24 6
28 17
42 26
38 30
21 16
11 1
27 38
27 33
20 2
2 45
25 42
40 27
15 32
41 37
8 14
20 43
2 2
8 22
3 24
36 39
8 26
38 15
30 31
45 17
22 25
18 9
40 5
32 29
1 43
26 33
4 37
42 2
14 31
24 4
40 23
10 45
39 32
26 15
6 38
33 21
2 36
31 13
1 9
10 21
25 10
30 18
6 19
43 42
33 30
30 36
15 5
13 27
5 8
38 34
2 9
25 6
8 17
27 45
5 2
30 7
11 43
26 13
6 27
31 20
1 10
3 4
25 1
18 6
37 8
13 18
34 15
7 18
36 34
21 39
40 13
44 34
6 45
4 1
11 13
39 7
32 1
7 27
35 16
8 9
36 2
31 1
38 23
15 15
22 24
29 31
21 30
8 40
17 11
44 41
2 21
26 24
31 45
32 9
35 28
8 6
42 43
12 15
12 2
6 26
9 36
12 37
41 39
2 17
15 6
33 31
22 35
15 20
2 14
19 4
35 44
3 29
45 9
29 36
24 42
15 28
32 32
15 33
43 3
32 25
26 38
42 20
35 11
29 41
5 45
45 13
22 10
13 20
17 18
38 35
10 4
17 8
18 14
45 35
17 28
18 21
26 6
19 42
42 3
8 37
5 1
16 22
13 24
24 10
16 41
18 33
40 40
42 24
26 17
20 34
40 43
37 21
28 7
30 29
32 7
38 33
2 18
25 17
24 12
36 9
25 8
38 16
33 11
43 10
21 1
21 26
29 21
16 23
36 43
5 30
9 4
10 25
21 13
22 22
17 24
16 10
14 12
4 6
28 19
35 40
8 23
4 25
17 29
37 37
31 8
11 16
33 39
43 30
21 37
7 36
44 24
26 8
15 44
8 4
6 3
31 16
36 21
33 34
22 15
27 31
24 17
13 5
5 14
30 42
21 14
28 9
22 18
24 21
41 25
41 42
29 22
3 1
11 4
32 37
4 29
28 10
39 34
19 35
3 19
35 5
12 16
34 11
12 13
26 5
9 30
43 36
24 16
19 1
34 7
22 34
18 44
42 41
11 28
3 32
25 38
24 37
45 14
10 17
21 5
6 5
11 18
42 5
21 45
14 30
41 16
43 40
20 23
15 1
35 15
44 2
4 42
18 39
41 10
7 41
14 42
35 17
38 6
11 34
4 15
27 11
32 31
10 35
26 3
5 19
24 26
45 39
30 28
11 44
40 29
25 7
16 40
11 38
31 28
6 32
27 1
8 36
11 30
25 13
26 41
10 38
14 22
44 26
25 35
4 14
43 38
44 1
24 7
1 34
40 45
22 20
22 3
22 2
18 20
36 42
38 3
26 10
29 15
26 23
24 29
11 20
7 15
15 21
34 39
4 35
35 43
9 41
45 22
31 42
29 17
32 15
1 42
20 11
17 5
34 13
25 16
22 45
5 18
17 35
20 39
1 30
44 42
2 23
21 24
4 9
45 6
18 45
39 33
44 15
39 38
35 23
20 9
17 37
16 31
28 30
13 29
21 31
34 25
3 7
38 25
17 25
30 38
27 42
18 2
11 39
16 19
18 7
33 37
5 22
28 15
36 40
1 40
9 24
10 26
34 37
1 23
39 5
40 22
24 39
27 23
4 45
35 7
21 3
2 11
16 35
12 27
37 34
36 10
9 9
20 15
26 27
25 45
13 41
15 38
21 44
23 18
41 13
10 5
6 37
17 23
36 36
22 38
43 27
2 43
41 12
21 20
18 23
33 26
27 32
40 24
23 6
16 12
29 32
18 42
2 27
42 45
30 27
6 25
31 36
22 44
7 22
10 18
1 39
10 16
4 36
39 26
14 7
4 39
5 21
39 37
1 7
37 38
10 12
35 32
18 12
16 42
4 22
3 36
2 30
35 22
14 21
37 18
40 31
29 5
25 32
35 20
15 13
18 18
15 2
6 1
1 26
37 3
29 18
40 18
19 17
7 43
6 15
35 36
2 6
36 6
21 40
5 42
44 36
24 23
1 31
12 28
42 8
34 20
31 25
27 39
29 14
44 33
39 16
25 29
40 3
42 19
21 15
10 43
32 34
31 37
22 6
23 39
3 8
14 29
25 20
28 36
12 34
14 6
13 37
42 23
25 23
23 1
28 42
43 43
3 17
19 13
6 36
24 34
17 42
41 36
13 9
1 38
28 35
39 31
15 25
19 32
17 2
12 19
13 43
13 33
8 5
35 26
7 44
37 5
41 38
32 2
19 18
37 39
9 6
11 9
21 8
34 29
25 40
16 9
31 27
12 23
8 28
24 30
9 29
36 38
23 38
34 19
4 20
3 20
19 33
30 17
14 16
21 22
39 41
10 3
32 36
14 2
26 9
4 31
33 38
35 12
42 16
6 12
9 35
8 42
28 18
11 15
19 34
32 10
22 37
22 41
26 19
38 4
22 9
4 27
2 5
24 27
14 41
37 7
26 43
28 40
38 12
24 5
29 38
1 12
41 29
22 13
32 19
5 34
45 42
37 15
37 1
21 32
40 4
12 20
12 10
20 3
41 2
41 27
41 6
1 21
32 26
1 2
21 43
42 17
33 15
29 20
35 18
22 5
8 3
23 17
16 28
15 36
14 10
45 5
37 2
9 37
18 29
18 27
15 23
17 4
21 12
29 34
38 9
17 6
26 16
14 33
12 43
37 44
26 2
23 12
28 43
28 20
22 32
13 23
41 32
3 41
4 34
5 40
19 43
7 33
27 20
10 9
6 39
4 11
33 9
35 41
16 32
8 2
6 14
27 21
45 7
31 3
12 33
30 10
29 23
3 12
7 7
24 13
2 33
13 10
23 41
4 19
14 34
25 4
42 1
8 12
6 9
16 20
5 7
23 15
8 38
31 21
3 40
30 41
17 17
30 13
45 32
10 30
7 11
43 39
41 17
16 3
29 26
7 16
20 16
23 35
8 7
38 31
7 5
40 19
27 2
34 17
45 40
29 35
16 29
38 8
6 24
11 27
9 2
42 40
32 17
34 22
41 5
15 22
42 39
33 25
25 43
40 11
33 28
20 1
17 13
23 14
25 21
34 28
36 8
8 44
23 8
10 20
45 10
27 34
25 25
18 34
9 27
25 5
11 31
34 26
4 41
14 3
11 40
27 43
20 30
32 13
13 2
13 15
23 5
35 3
13 31
7 14
21 17
9 12
33 14
32 4
40 32
43 17
18 43
31 11
45 8
8 39
3 3
42 25
15 10
35 37
19 40
27 44
25 30
5 41
29 16
22 30
33 43
33 6
39 35
44 6
2 7
41 35
34 2
31 35
5 38
1 4
3 38
9 13
36 3
25 44
45 33
9 25
26 44
27 22
40 35
28 22
4 32
41 22
42 31
40 6
12 39
16 27
9 44
24 22
8 25
43 29
12 40
8 30
45 12
43 24
14 1
38 13
33 12
32 42
10 19
40 21
29 27
36 44
34 30
25 34
35 21
27 24
12 9
23 30
12 12
6 43
2 34
1 6
12 21
7 26
2 29
15 41
33 4
37 43
22 28
36 22
11 41
34 4
34 36
31 29
40 1
18 13
13 6
41 3
19 28
20 14
32 30
32 39
44 12
25 12
44 20
9 19
9 23
21 35
1 28
45 29
19 22
10 34
11 19
42 27
37 4
45 20
19 16
31 33
32 40
19 23
12 7
3 45
42 37
42 6
34 31
30 39
7 28
43 12
18 35
11 10
22 39
21 9
38 29
19 25
9 43
39 12
7 34
36 31
38 2
9 42
44 17
12 18
38 45
39 3
11 6
10 42
34 27
33 16
39 13
5 26
13 35
23 13
25 2
11 5
36 16
27 17
22 29
6 2
15 35
42 14
6 41
28 16
17 14
5 28
31 10
43 31
36 15
1 1
27 15
33 22
3 25
37 11
24 3
2 41
38 28
27 4
24 24
6 40
19 36
44 39
33 18
29 4
7 32
45 23
31 22
38 38
9 11
9 33
16 14
15 9
17 26
45 3
19 38
32 18
32 23
16 24
13 32
16 18
40 42
1 44
34 6
30 14
25 36
40 39
29 44
27 7
30 22
43 45
13 34
25 11
26 14
36 45
20 18
34 9
11 26
27 3
23 26
5 17
21 11
28 13
15 31
40 44
33 42
24 38
18 40
4 16
19 45
31 9
25 15
20 20
30 25
32 14
8 19
14 38
1 37
9 40
20 24
16 26
25 9
35 33
2 20
44 35
19 10
44 40
30 24
37 45
12 4
24 9
29 7
23 25
35 34
29 28
42 36
11 3
11 23
7 37
28 2
17 3
13 36
39 29
20 40
33 5
13 25
32 38
10 22
8 13
5 16
9 5
36 12
6 33
15 26
25 3
42 4
20 13
16 44
2 31
2 15
14 9
11 12
42 29
40 17
10 8
40 2
6 22
10 33
14 24
43 19
17 38
26 1
28 21
3 35
30 16
15 7
26 22
26 31
18 28
19 19
22 7
34 12
4 2
41 4
13 40
22 8
14 5
14 44
27 19
14 14
23 44
17 32
33 32
39 9
40 10
31 19
30 6
3 21
22 36
28 4
1 41
44 29
30 4
9 34
7 8
6 13
24 41
41 19
8 35
24 45
15 19
15 8
38 43
27 12
11 8
4 44
19 44
22 11
33 8
33 23
35 27
2 25
6 11
15 45
19 5
20 36
42 15
21 7
23 45
5 32
24 1
25 41
18 3
33 19
33 27
15 17
9 31
13 38
10 1
43 6
5 3
8 34
16 7
15 40
42 33
14 26
11 21
21 2
22 40
20 45
24 35
27 30
34 1
28 39
45 30
32 45
34 8
33 41
2 26
39 20
42 18
21 18
10 40
35 2
4 17
11 22
6 29
19 7
9 17
30 5
35 1
37 41
9 15
4 38
7 35
18 25
24 15
35 31
36 14
22 23
44 9
37 10
20 29
6 4
2 28
13 3
3 30
9 18
25 18
14 8
36 35
23 10
1 33
12 1
42 13
45 26
35 4
20 28
6 31
37 35
27 25
36 25
25 39
27 29
2 13
12 17
15 39
28 3
15 3
35 14
34 43
19 8
34 45
40 34
39 21
28 14
20 37
29 11
38 11
33 7
14 4
41 11
9 26
18 10
1 24
22 27
35 29
11 37
1 45
13 45
17 31
13 42
16 5
5 44
24 44
45 1
30 15
29 3
32 44
1 35
21 23
25 24
44 3
10 11
42 35
31 41
41 31
30 1
40 26
7 30
18 37
39 8
35 39
45 18
16 4
40 38
17 36
31 40
16 34
16 38
31 34
25 19
24 36
5 5
27 41
34 14
13 30
32 6
35 24
32 28
33 3
14 40
2 4
15 16
36 13
4 30
13 17
44 31
38 18
3 15
22 42
17 41
12 44
34 16
5 33
1 14
4 23
35 45
13 19
44 18
12 36
28 28
12 45
7 20
5 39
30 9
21 36
20 27
42 38
17 45
28 41
43 32
21 21
6 34
14 13
7 10
41 41
39 30
19 26
9 14
35 13
12 25
10 23
39 42
9 3
18 36
18 24
31 15
38 10
34 18
19 2
12 29
27 5
37 32
43 37
3 16
1 22
32 43
4 26
7 31
18 30
27 13
23 3
7 21
20 6
2 44
30 34
5 35
33 20
17 12
20 12
17 22
5 11
22 4
16 37
45 25
16 30
17 16
5 25
29 8
42 44
24 40
19 14
2 35
24 33
39 39
10 39
23 34
16 21
12 22
19 24
15 24
38 5
4 40
22 14
10 28
44 11
1 20
36 5
29 2
23 7
15 30
33 10
5 27
44 45
7 4
30 44
14 19
36 20
5 37
39 45
23 29
3 18
5 24
6 35
43 23
43 33
30 35
8 11
20 33
4 4
32 33
5 20
45 4
3 2
26 45
19 12
31 32
40 25
42 32
1 5
22 19
40 12
2 42
43 5
32 16
26 30
43 14
11 25
14 17
8 24
38 24
44 13
27 14
39 27
36 28
43 16
37 22
11 24
38 7
36 37
39 23
34 34
16 17
13 21
7 17
40 9
39 4
31 4
8 29
5 31
6 18
15 37
43 25
15 34
7 1
40 16
1 32
44 8
20 44
45 24
19 6
15 12
31 31
31 2
35 19
9 1
1 36
25 22
6 7
7 39
10 14
12 41
19 9
3 5
29 42
3 43
23 40
40 30
37 25
8 1
40 20
3 28
8 45
3 22
43 20
7 23
27 37
35 42
32 5
41 8
36 4
4 5
5 23
20 10
7 25
13 1
44 22
23 24
16 6
28 5
37 6
23 28
39 22
9 39
33 33
20 32
16 25
10 7
39 18
29 1
44 16
6 20
2 3
3 6
30 2
43 26
10 6
3 44
7 42
28 34
38 14
3 27
38 41
17 9
45 44
2 32
9 7
44 7
14 45
23 43
2 12
9 21
18 19
23 9
22 33
5 36
25 26
28 27
2 40
20 38
12 24
23 22
29 25
39 25
36 24
9 8
3 39
22 16
24 20
36 33
6 16
40 36
28 37
17 10
32 3
23 4
11 2
30 8
14 32
4 24
45 38
42 9
38 32
18 32
26 28
23 42
37 17
24 31
4 43
4 21
45 31
44 32
44 10
2 38
14 43
17 15
8 32
44 44
40 28
38 42
3 33
20 42
22 17
7 13
15 4
1 13
6 8
36 19
35 35
39 36
32 41
28 29
41 21
29 39
37 31
23 32
8 8
24 19
4 10
13 28
16 2
28 45
41 18
37 16
43 1
19 39
33 17
18 11
21 19
27 35
41 24
28 32
44 19
37 12
1 18
1 15
41 44
39 17
24 8
27 27
41 43
