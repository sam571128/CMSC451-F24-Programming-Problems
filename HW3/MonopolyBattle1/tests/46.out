85 112 62 62 62 62 62 62 106 108 62 62 62 62 62 85 82 62 62 144 62 62 62 69 62 62 62 62 108 62 62 70 62 134 62 62 62 62 62 62 174 62 62 78 62 